Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -162,7 +162,7 @@ open class ComposerVC: _ViewController,
                 editingMessage: message,
                 quotingMessage: nil,
                 threadMessage: threadMessage,
-                attachments: attachments,
+                attachments: message.allAttachments.toAnyAttachmentPayload,
```

Comment:
This is the only concerning part since it looks like if the customer has custom attachments, it won't be able to edit them, right? 🤔 Can we provide a hook somewhere where customers can provide their own attachment payloads decoding logic?